CHANGELOG.md

```diff
@@ -114,13 +114,26 @@ Solidity optimizer: `disabled`
 -   `MultiSendCallOnly` at `0x9641d764fc13c8B624c04430C7356C1C7C8102e2`
 -   `CreateCall` at `0x9b35Af71d77eaf8d7e40252370304687390A1A52`
 -   `SignMessageLib` at `0xd53cd0aB83D845Ac265BE939c57F53AD838012c9`
+-   `SafeToL2Setup` at `0x80E0d1577aD3d982BF2F49aAB00BfA161AA763c4`
```

Comment:
This is also missing `SafeMigration` and `SafeToL2Migration` library contracts.

Reply:
https://github.com/safe-global/safe-smart-account/pull/816/commits/4460f0b28664d0323364fda8e77d01b073043f43